releasenotes/notes/apm-sampling-priority-stats-extension-68de07f56a23e7f9.yaml

```diff
@@ -0,0 +1,7 @@
+---
+enhancements:
+  - |
+    Now we report counts of how many traces per priority we receive for all priority codes between -10 and 10.
+fixes:
+  - |
+    Count of traces with priority 2 (PriorityUserKeep) were not being updated correctly.
```

Comment:
I don't think we want a release note on an internal debugging metric, especially that not all values or documented or exist yet
To pass the CI without release note the tag `changelog/no-changelog` can be added

Reply:
Makes sense